website/cue/reference/components/sources/kubernetes_logs.cue

```diff
@@ -42,6 +42,11 @@ components: sources: kubernetes_logs: {
 			"""
 				[Kubernetes](\(urls.kubernetes)) version `\(services.kubernetes.versions)` is required.
 				""",
+			"""
+				This source reads log files from the host, *not* the Kubernetes API. If Vector is running within Kubernetes,
```

Comment:
I don't think running _outside_ Kubernetes is realistic for the source to be honest - or at least I can't think of a sensible way of configuring that.

Reply:
I could see it happening if someone runs a super manual setup, but it isn't the typical path these days.

Reply:
I'm not sure how you'd have access to the filesystem in kubernetes from a location outside?

Reply:
Ah, running on the Kubernetes Node, but not in the cluster itself would be possible. Overall I think this sounds more suited to the `How it Works` part of the docs than requirement

Reply:
I actually think that this is a requirement: if you don't have access to the logs dir from within Vector, you can't really read any logs. This'll almost always involve specific `hostPath` volume configuration. On the other hand, I feel like "how it works" is more of a passive thing, with info that adds context, but doesn't inform the setup.